packages/tools/client-debug-view/.mocharc.js

```diff
@@ -0,0 +1,12 @@
+/*!
```

Comment:
Unnecessary? Doesn't seem like we're doing mocha tests or referencing this file. If so, a few dependencies in `package.json` could also go away.

Reply:
I am intending to add tests, but I think they will end up using Jest rather than Mocha, so that's a good call.